scribus/ui/alignselect.cpp

```diff
@@ -110,3 +110,7 @@ void AlignSelect::languageChange()
 	TextB->setToolTip( tr("Align Text Justified"));
 	TextF->setToolTip( tr("Align Text Forced Justified"));
 }
+void AlignSelect::changeIcon(QToolButton* button, QString iconName)
+{
+	button->setIcon(IconManager::instance()->loadIcon(iconName));
```

Comment:
It should be `textAlignment->TextB->setIcon(IconManager::instance()->loadIcon("16/format-justify-fill-block-right.png"));`, why wouldn’t this work. We should at least find the reason.